surveyor/jetstream.go

```diff
@@ -198,7 +198,7 @@ func NewJetStreamAdvisoryListener(f string, sopts Options) (*JSAdvisoryListener,
 
 	sopts.Name = fmt.Sprintf("%s (jetstream %s)", sopts.Name, opts.AccountName)
 	sopts.Credentials = opts.Credentials
-	nc, err := connect(&sopts, fmt.Sprintf("%s JetStream %s", sopts.Name, opts.AccountName))
```

Comment:
there were handled elsewhere already leading to names like `NATS_Surveyor - 2786262410e5 (jetstream RI JetStream) JetStream RI JetStream`